bigtable-dataflow-parent/bigtable-beam-import/README.md

```diff
@@ -1,44 +1,204 @@
-    --destinationPath=gs://[bucket_name]/[export_directory]/ \
-    --tempLocation=gs://[bucket_name]/[temp_work_directory]/ \
-    --maxNumWorkers=[10x number of nodes] \
+    --project=$PROJECT_ID \
+    --bigtableInstanceId=$INSTANCE_ID \
+    --bigtableTableId=$TABLE_NAME \
+    --destinationPath=$BUCKET_NAME/hbase_export/ \
+    --tempLocation=$BUCKET_NAME/hbase_temp]/ \
+    --maxNumWorkers=$(expr 10 \* $CLUSTER_NUM_NODES) \
 ```
 
-## Import
 
-Create the table in your cluster.
+## Importing to Bigtable
+
+This folder contains pipelines to help import data via snapshots or sequence files
+and can be used for HBase or Bigtable. Before you begin your import you must create
+the tables in Bigtable via the [schema translation tool](bigtable-hbase-1.x-parent/bigtable-hbase-tools/README.md)
+or using the Bigtable command line tool and running `cbt createtable your-table-name`.
+Once your import is completed follow the instructions for the validator below to ensure it was successful.
+
+Please pay attention to the Cluster CPU usage and adjust the number of Dataflow workers accordingly.
+
+### Snapshots (preferred method)
 
-On the command line:
+Set the environment variables.
+
+```
+PROJECT_ID=your-project-id
+INSTANCE_ID=your-instance-id
+TABLE_NAME=your-table-name
+REGION=us-central1
+
+SNAPSHOT_PATH="gs://your-bucket/hbase-migration-snap"
+SNAPSHOT_NAME=your-snapshot-name
+```
+
+Run the import.
+```
+java -jar bigtable-beam-import-1.14.1-SNAPSHOT-shaded.jar importsnapshot \
+    --runner=DataflowRunner \
+    --project=$PROJECT_ID \
+    --bigtableInstanceId=$INSTANCE_ID \
+    --bigtableTableId=$TABLE_NAME \
+    --hbaseSnapshotSourceDir=$SNAPSHOT_PATH/data \
+    --snapshotName=$SNAPSHOT_NAME \
+    --stagingLocation=$SNAPSHOT_PATH/staging \
+    --tempLocation=$SNAPSHOT_PATH/temp \
+    --region=$REGION
+```
+
+
+### Sequence Files
+
+Set the environment variables.
+
+```
+PROJECT_ID=your-project-id
+INSTANCE_ID=your-instance-id
+CLUSTER_NUM_NODES=3
+CLUSTER_ZONE=us-central1-a
+TABLE_NAME=your-table-name
 
+BUCKET_NAME=gs://bucket-name
+```
+
+Run the import.
 ```
 java -jar bigtable-beam-import-1.14.1-shaded.jar import \
     --runner=dataflow \
-    --project=[your_project_id] \
-    --bigtableInstanceId=[your_instance_id] \
-    --bigtableTableId=[your_table_id] \
-    --sourcePattern='gs://[bucket_name]/[import_directory]/part-*' \
-    --tempLocation=gs://[bucket_name]/[temp_work_directory] \
-    --maxNumWorkers=[3x number of nodes] \
-    --zone=[zone of your cluster]
+    --project=$PROJECT_ID \
+    --bigtableInstanceId=$INSTANCE_D \
+    --bigtableTableId=$TABLE_NAME \
+    --sourcePattern='$BUCKET_NAME/hbase-export/part-*' \
+    --tempLocation=$BUCKET_NAME/hbase_temp \
+    --maxNumWorkers=$(expr 3 \* $CLUSTER_NUM_NODES)  \
+    --zone=$CLUSTER_ZONE
+```
+
+
+## Validating data
+
+Once your snapshot or sequence file is imported, you should run the validator to
+check if there are any rows with mismatched data. 
+
```

Comment:
Before providing commands to run syncTable, we should provide a command to run HashTable on HBase and say that output of that job goes into `    --hashTableOutputDir=$SNAPSHOT_PATH/hashtable \` parameter.

Reply:
ah yes and I needed to show exporting it to a bucket too. fixed